warehouse/redshift/redshift.go

```diff
@@ -354,7 +354,7 @@ func (rs *HandleT) loadTable(tableName string, columnMap map[string]string) (err
 		sortedColumnNames += fmt.Sprintf(`"%s"`, key)
 	}
 
-	stagingTableName := fmt.Sprintf(`%s%s-%s`, stagingTablePrefix, tableName, uuid.NewV4().String())
+	stagingTableName := misc.TruncateStr(fmt.Sprintf(`%s%s-%s`, stagingTablePrefix, tableName, uuid.NewV4().String()), 127)
```

Comment:
Better to move uuid before table name, because of the truncation. A large table name might remove uuid completely.